FileLiberator/DownloadBook.cs

```diff
@@ -40,7 +41,53 @@ private static string getDownloadPath(LibraryBook libraryBook)
 				libraryBook.Book.Title,
 				"aax",
 				libraryBook.Book.AudibleProductId);
+		private async Task<string> downloadAacxBookAsync(LibraryBook libraryBook, string tempAaxFilename)
```

Comment:
1) typo: aacx should be aaxc -- the last 2 letters are flipped

2) These new AudibleKey and AudibleIV are new to me. Are all audible books now encoded per book rather than per account activation key? Is this true for both aax and aaxc? Also, if true for both types, we can just reuse the downloadBookAsync name so it doesn't appear that it works only for non-aax